README.md

```diff
@@ -1,2 +1,182 @@
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby (v3.2.2)</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails (v7.0.5)</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://postgresql.org">PostgreSQL</a></li>
+  </ul></details>
+
+# Description
+
+> You can watch a demo of the video description below.
+
+[Link to Loom.](https://www.loom.com/share/853f6873d4af4481b708199f2abf384d?sid=79aa4915-9a24-4f12-94a3-2cec7d81e365)
+
+### Key Features <a name="key-features"></a>
+
+- Registered users will be able to create new category.
+
+- Users will be able to create a transaction entity from the category chosen
+
+- Users will be able to go back to home.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. Ruby installed on your machine
+2. Text editor (VSCode, Sublime, Atom, etc.)
+3. Install Rails gem.
+4. PostgreSQL running on your machine
+
+### Setup
+
+Clone this [repository](git@github.com:c00p75/budget-rails.git) to your desired folder:
+
+```sh
+  git clone https://github.com/calebchris000/Budget-App.git
+  cd Budget-App
+```
+
+### Dependencies
+
+Install Dependencies with:
+
+```sh
+  bundle install
+```
+
+create database in your postgresql with:
+
+```sh
+  rails db:create
+```
+
+> Note: If you have any problem with the database, you can check the database.yml file and change the username and password to your own.
+
+Run migration with:
+
+```sh
+  rails db:migrate
+```
+
+### Usage
+
+Run Project with:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+Run tests with:
+
+```sh
+  rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Caleb Nwaizu**
+
+- GitHub: [@calebchris000](https://github.com/calebchris000)
+- LinkedIn: [@calebchris000](https://www.linkedin.com/in/calebchris000/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add Authentication page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/c00p75/budget-rails/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please leave a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for providing the reading materials that aided me during the project development
```

Comment:
- [ ] It would be better if we give appropriate credit to the author of the original design
![Screenshot from 2023-07-04 23-12-44](https://github.com/calebchris000/Budget-App/assets/46388113/4e08ba19-6357-4cc7-9e13-9dd2384b2af0)